### DISCUSSION TITLE
Is this app still maintain?

### DISCUSSION THREAD

**Discussion by HitMannCU** [activexray/Doplarr#130] [General]
Just asking, because the last update was like almos a year ago

**Comment by activexray** [comment]
I fix the bugs that I can - and everyone’s issues for about the last year have been in people’s setup, discord bugs I can’t fix, or things I don’t know how to fix. The bot works fine for me, so as far as I’m concerned there’s nothing to do. I don’t have the time, motivation, or interest in adding new features right now. Please remember, I put this work out for free.

**Reply by mia-riezebos** [reply]
your work is highly appreciated and the bot is working perfectly fine in my media stack!
